packages/faceted-search/src/queryClient/tql.js

```diff
@@ -9,7 +9,7 @@ const getBadgeQueryValues = ({ properties }) => [
 
 const getBadgesQueryValues = badges => badges.map(getBadgeQueryValues);
 
-const isNotEmptyOrNull = value => value && value.length;
+const isNotEmptyOrNull = value => typeof value === 'number' || (value && value.length);
```

Comment:
do not change this one. I prefer you to add an other isNumber, add some test (using NaN for example).
This one is used to check ids, ... not only the tql value field.

Reply:
@jmfrancois Thx, you're right, i introduced a bug. Just fix that with a split function and more tests in `packages/faceted-search/src/queryClient/tql.test.js`. Also exclude `NaN` from condition.